server/src/routes/api/v1/usersRouter.js

```diff
@@ -22,7 +25,41 @@ usersRouter.post("/", async (req, res) => {
     throw error;
   }
 
+  // const fuzziesAsElevator = await newUser.$relatedQuery("givenWarmFuzzies")
+
   return res.status(201).json({ user: newUser });
 });
 
+usersRouter.get("/:id/warm-fuzzies/given", async (req, res) => {
+  const user = await User.query().findById(req.params.id);
+  const elevations = await WarmFuzzy.query().select('createdAt', 'title', 'description', 'recipientId', 'elevatorId').where('elevatorId', `${user.id}`);
+
+  const promises = elevations.map(fuzzy => {
+    return givenFuzzySerializer(fuzzy)
+  })
+  const serializedFuzzies = await Promise.all(promises)
+
+  return res?.status(201).json(serializedFuzzies);
+})
+
+usersRouter.get("/:id/warm-fuzzies/received", async (req, res) => {
+  const user = await User.query().findById(req.params.id);
+  const elevations = await WarmFuzzy.query().select('createdAt', 'title', 'description', 'recipientId', 'elevatorId').where('recipientId', `${user.id}`);
```

Comment:
I would probably avoid the explicit select here.

Reply:
consider using `$relatedQuery` 

```javascript
const elevations = await user.$relatedQuery("givenWarmFuzzies")
```

the other thing to keep in mind here is that your elevations are unconstrained. We probably want to paginate this data because the list of awarded and received fuzzies will grow over time.